modules/reference/pages/cluster-properties.adoc

```diff
@@ -3031,7 +3018,7 @@ Max number of the active sessions (producers). When the threshold is passed Redp
 
 === max_in_flight_pandaproxy_requests_per_shard
 
-Maximum number of in flight HTTP requests permitted in pandaproxy per shard.  Any additional requests above this limit will be rejected with a 429 error.
+Maximum number of in-flight HTTP requests to HTTP Proxy permitted per shard.  Any additional requests above this limit will be rejected with a 429 error.
```

Comment:
ENG: We should standardize the descriptions to at least "HTTP Proxy" (and ideally the property names would be updated from pandaproxy as well)